### PR TITLE
fix #61894 news.thewindowsclub.com

### LINKED ISSUE #61894: news.thewindowsclub.com
### Issue URL (Ads)
 [https://news.thewindowsclub.com/is-this-what-the-windows-10-start-menu-could-look-like-102382/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fnews.thewindowsclub.com%2Fis-this-what-the-windows-10-start-menu-could-look-like-102382%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Windows v7.5 nightly 21
Browser: | Edge 84.0.522.61 (Chromium-based)
AdGuard driver: | WFP
Stealth mode options: | Hide your search queries, <br/>Send Do-Not-Track header, <br/>Remove X-Client-Data header from HTTP requests, <br/>Strip URLs from tracking parameters, <br/>Self-destructing third-party cookies (180), <br/>Block WebRTC, <br/>Block Push API, <br/>Block Location API, <br/>Block Flash
Filters: | <b>Ad Blocking:</b><br/> Peter Lowe's Blocklist, <br/>AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> I don't care about cookies, <br/>AdGuard Annoyances<br/><br/><b>Security:</b><br/> Malware Domains Blocklist, <br/>Spam404<br/><br/><b>Other:</b><br/> AdGuard Experimental<br/><br/><b>Language-specific:</b><br/> Official Polish filters for AdBlock, uBlock Origin & AdGuard, <br/>Polskie Filtry Ciasteczkowe, <br/>KAD - Przekręty, <br/>Polish Anti Adblock Filters, <br/>AlleBlock, <br/>EasyList Polish
Custom filter subscriptions: | https://gitlab.com/ZeroDot1/CoinBlockerLists/raw/master/list_browser_UBO.txt,<br>https://raw.githubusercontent.com/PolishFiltersTeam/PolishAnnoyanceFilters/master/PAF_pop-ups.txt,<br>https://raw.githubusercontent.com/PolishFiltersTeam/KAD/master/KADfake.txt

### PR DESCRIPTION
**Pull request by stanislav-atr**
#61894

### CODE REVIEW

**Review by Sergey-Lyapin — CHANGES_REQUESTED** [review]
`thewindowsclub.com###ss-floating-bar`
It's better to move this rule to popups.txt

**Comment by Alex-302** [comment]
> It's better to move this rule to popups.txt

not popup

File: AnnoyancesFilter/sections/annoyances.txt [hunk 1]
```diff
@@ -529,6 +529,8 @@ gnavi.co.jp##.hatena-module-body .hatena-fotolife[alt][width]
 ! https://github.com/AdguardTeam/AdguardFilters/issues/27911
 closermag.fr#$#body[style*="overflow: hidden;"] { overflow: visible!important; }
 closermag.fr#$#html[style^="overflow: hidden;"] { overflow: visible!important; }
+! https://github.com/AdguardTeam/AdguardFilters/issues/61894
+thewindowsclub.com###ss-floating-bar
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/annoyances.txt:533]: to social filter

**Review by Alex-302 — CHANGES_REQUESTED** [review]

**Comment by stanislav-atr** [comment]
> 
> 
> leftover

Nothing there with or w/o adguard.

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**